### PR TITLE
Add JPA CDI Test case to JPA 22 Test Bucket

### PR DESCRIPTION
**Pull request by jgrassel**
Signed-off-by: Joe Grassel <jgrassel@us.ibm.com>

Adding a FAT to com.ibm.ws.jpa_22_fat that tests JPA 2.2 + CDI 2.0 integration with EntityListeners.

### CODE REVIEW

**Comment by jgrassel** [comment]
#build

**Comment by jgrassel** [comment]
Failed test in my personal build was com.ibm.ws.app.manager.wab.configure.fat.ConfigurableWABTests.	
testConflictConfiguration() - does not appear to be related to my change.

File: dev/com.ibm.ws.jpa_22_fat/bnd.bnd [hunk 1]
```diff
@@ -28,6 +30,7 @@ runtime.minimum.java.level: 1.8
 # Dependencies may be local bundles (e.g. com.ibm.websphere.javaee.servlet.3.1)
 #      or binaries from Artifactory (e.g. org.apache.derby:derbynet)
 -buildpath: \
+    com.ibm.websphere.javaee.cdi.2.0;version=latest,\
 	com.ibm.websphere.javaee.servlet.3.1;version=latest,\
```
Comment (tevans78) [inline dev/com.ibm.ws.jpa_22_fat/bnd.bnd]: I'd recommend going with servlet-4.0

File: dev/com.ibm.ws.jpa_22_fat/bnd.bnd [hunk 2]
```diff
@@ -16,10 +16,12 @@ javac.source: 1.8
 
 src: \
 	fat/src,\
+	test-applications/cdi/src,\
 	test-applications/jpa21bootstrap/src,\
 	test-applications/jpa22bootstrap/src
 
 test.project: true
+tested.features=cdi-2.0
```
Comment (daliasheasha) [inline dev/com.ibm.ws.jpa_22_fat/bnd.bnd]: tested.features=cdi-2.0 should be tested.features:cdi-2.0

Comment (jgrassel) [reply dev/com.ibm.ws.jpa_22_fat/bnd.bnd]: Done.

**Comment by daliasheasha** [comment]
Thanks for making the review comment updates. I'll go ahead and approve after another build is submitted to make sure the FAT changes are still good.

**Comment by jgrassel** [comment]
#build

**Comment by jgrassel** [comment]
Failures look jsf + opentracing related, not JPA.

**Pull request merged**